pkgs/data/fonts/cozette/default.nix

```
@@ -9,16 +9,22 @@ stdenvNoCC.mkDerivation rec {
     hash = "sha256-v1UWrVx1PnNPiFtMMy4kOkIe//iHxx0LOA4nHo95Zws=";
   };
 
+  buildInputs = [ psftools ];
+
   installPhase = ''
     runHook preInstall
 
     install -Dm644 *.ttf -t $out/share/fonts/truetype
     install -Dm644 *.otf -t $out/share/fonts/opentype
     install -Dm644 *.bdf -t $out/share/fonts/misc
     install -Dm644 *.otb -t $out/share/fonts/misc
+    install -Dm644 *.fnt -t $out/share/fonts/misc
     install -Dm644 *.woff -t $out/share/fonts/woff
     install -Dm644 *.woff2 -t $out/share/fonts/woff2
 
+    for f in *.fnt; do fnt2psf $f $(basename -- $f .fnt).psf; done
+    install -Dm644 *.psf -t $out/share/fonts/misc
```

How about using consolefonts for psf path as other packages?

```suggestion
    install -Dm644 *.psf -t $out/share/consolefonts
```